src/pages/api/file/upload-file.ts

```diff
@@ -133,7 +133,7 @@ const videoLogic = async (
 		mediaType: "",
 		iframeAllowed: false,
 		isPageScreenshot: null,
-		video_url: thumbnailUrl?.publicUrl ?? null,
+		video_url: null,
```

Comment:
@B-a-l-aj-i You removed the thumbnailUrl? Don't need to send it in the API response?

Reply:
So, `ogImage` column has the thumbnail. @Karthik-B-06 You might want to consume the `ogImage` column for thumbnail.